getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -138,9 +138,9 @@ SELECT * FROM timescaledb_information.continuous_aggregates;
 **Results:**
 
 ```bash
-hypertable_schema|hypertable_name |view_schema|view_name     |view_owner|materialized_only|compression_enabled|materialization_hypertable_schema|materialization_hypertable_name|view_definition                                                                                                                                                                                                                                                |
------------------+----------------+-----------+--------------+----------+-----------------+-------------------+---------------------------------+-------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-public           |stocks_real_time|public     |candle_one_min|tsdbadmin |false            |false              |_timescaledb_internal            |_materialized_hypertable_3     | SELECT time_bucket('00:01:00'::interval, stocks_real_time."time") AS bucket,¶    stocks_real_time.symbol,¶    first(stocks_real_time.price, stocks_real_time."time") AS open,¶    max(stocks_real_time.price) AS high,¶    min(stocks_real_time.price) AS low,|
+hypertable_schema|hypertable_name |view_schema|view_name              |view_owner|materialized_only|compression_enabled|materialization_hypertable_schema|materialization_hypertable_name|view_definition                                                                                                                                                                           |
+-----------------+----------------+-----------+-----------------------+----------+-----------------+-------------------+---------------------------------+-------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+public           |stocks_real_time|public     |stock_candlestick_daily|tsdbadmin |false            |false              |_timescaledb_internal            |_materialized_hypertable_3     | SELECT time_bucket('1 day'::interval, srt."time") AS day,¶    srt.symbol,¶    max(srt.price) AS high,¶    first(srt.price, srt."time") AS open,¶    last(srt.price, srt."time") AS close,|
```

Comment:
```suggestion
public           |stocks_real_time|public     |stock_candlestick_daily|tsdbadmin |f            |f             |_timescaledb_internal            |_materialized_hypertable_3     | SELECT time_bucket('1 day'::interval, srt."time") AS day,¶    srt.symbol,¶    max(srt.price) AS high,¶    first(srt.price, srt."time") AS open,¶    last(srt.price, srt."time") AS close,|
```

Reply:
Thank you @Rajakavitha1 
I took into account this suggestion but had to apply it manually cause it takes additional offset when cutting off the part of the string.